templates/quickstart-jira-dc.template.yaml

```diff
@@ -939,6 +948,7 @@ Resources:
             Action: ['sts:AssumeRole']
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM
+        - arn:aws:iam::aws:policy/AmazonEC2ReadOnlyAccess
```

Comment:
Why do we have this policy? I cannot see anything in this template nor `dc-deployments-automation` PR that would need it. Looks like `CloudWatchAgentServerPolicy` should be enough per https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/create-iam-roles-for-cloudwatch-agent-commandline.html
Apologies if I have missed something.

Reply:
It was actually mentioned in an official AWS blog post, but it might be out of date. I'll try with that instead.